### PR TITLE
back up uptime.com

### PR DESCRIPTION
**Pull request by w8896699**

### CODE REVIEW

File: uptime/monitor-setup/checks.json [hunk 1]
```diff
@@ -469,31 +465,86 @@
...
+      "msp_proxy": "",
+      "msp_dns_server": "",
+      "msp_dns_record_type": "",
+      "msp_status_code": "",
+      "msp_send_string": "",
+      "msp_expect_string": "\"finished\":true",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "SSL_TLS",
+      "msp_threshold": 40,
+      "msp_headers": "X-N8N-API-KEY: n8n_api_1a70d4baeef73d67f0b98e2e8a40095f716b044db7ea701dd5862d1c2727331cf8a4312b2576f999",
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:518]: this is a public repo, please remove sensitive info from the payload! you can keep them in the team shared Vault space!

File: uptime/monitor-setup/checks.json [hunk 2]
```diff
@@ -506,22 +557,22 @@
...
       "msp_dns_record_type": "",
       "msp_status_code": "",
       "msp_send_string": "",
-      "msp_expect_string": "True",
+      "msp_expect_string": "\"finished\":true",
       "msp_expect_string_type": "STRING",
       "msp_encryption": "SSL_TLS",
       "msp_threshold": 40,
-      "msp_headers": "",
+      "msp_headers": "X-N8N-API-KEY: n8n_api_1a70d4baeef73d67f0b98e2e8a40095f716b044db7ea701dd5862d1c2727331cf8a4312b2576f999",
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:575]: same here!

File: uptime/monitor-setup/checks.json [hunk 3]
```diff
@@ -565,22 +614,98 @@
...
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:632]: same here!

File: uptime/monitor-setup/checks.json [hunk 4]
```diff
@@ -565,22 +614,98 @@
...
+      "msp_proxy": "",
+      "msp_dns_server": "",
+      "msp_dns_record_type": "",
+      "msp_status_code": "",
+      "msp_send_string": "",
+      "msp_expect_string": "\"finished\":true",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "SSL_TLS",
+      "msp_threshold": 40,
+      "msp_headers": "X-N8N-API-KEY: n8n_api_1a70d4baeef73d67f0b98e2e8a40095f716b044db7ea701dd5862d1c2727331cf8a4312b2576f999",
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:708]: same here!

File: uptime/monitor-setup/checks.json [hunk 5]
```diff
@@ -590,21 +715,248 @@
...
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:765]: same here!

File: uptime/monitor-setup/checks.json [hunk 6]
```diff
@@ -590,21 +715,248 @@
...
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:822]: same here!

File: uptime/monitor-setup/checks.json [hunk 7]
```diff
@@ -590,21 +715,248 @@
...
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:879]: same here!

File: uptime/monitor-setup/checks.json [hunk 8]
```diff
@@ -590,21 +715,248 @@
...
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:936]: same here!

File: uptime/monitor-setup/checks.json [hunk 9]
```diff
@@ -620,14 +972,73 @@
...
+      "msp_expect_string": "",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "",
+      "msp_threshold": 60,
+      "msp_headers": "",
+      "msp_script": "[{\"step_def\":\"C_OPEN_URL\",\"values\":{\"url\":\"https://registry.developer.gov.bc.ca\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\"button\",\"text\":\"Login\",\"is_regex\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".css-1jhnpo2\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT\",\"values\":{\"element\":\"#password\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#user\",\"text\":\"psta\",\"typing_delay\":\"\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#password\",\"text\":\"p1aTf0rmS3rV!c3sT3aM\",\"typing_delay\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".login-form-action > .btn-primary\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\".css-lgt0d3 .css-vurnku > .css-vurnku\",\"text\":\"Git documentation for Platform Services Registry\",\"is_regex\":\"\"}}]",
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/checks.json:994]: same here!

Comment (ShellyXueHan) [reply uptime/monitor-setup/checks.json:994]: you might need to request to update the IDIR account secret just in case!

**Review by ShellyXueHan — CHANGES_REQUESTED** [review]
also in general, please assign the specific contacts for the new checks instead of just using `default`. For the PUT checks, I think it would be great to group them separate from `shared-services`

File: uptime/monitor-setup/checks.json (hunk 9 above)
Comment (NickCorcoran) [reply uptime/monitor-setup/checks.json:994]: You will definitely need to change the IDIR account password ASAP!  As Shelly indicated, secrets should only live in:
- Vault
-  GitHub Secrets, or 
- OCP secrets (only if not possible to store in other locales).

Comment (rstens) [reply uptime/monitor-setup/checks.json:994]: Also the N8N API key + the url completely opens up N8N

**Pull request closed without merging**